### ISSUE TITLE
Demo throws `AttributeError: ACTION_CLASSIFICATION` on detection model

### ISSUE THREAD

**Issue by QSmally** [open-edge-platform/training_extensions#2061]
**Describe the bug**

I trained a model using `otx train` and exported it with `otx export` as well as created the demo instance. Like with training, I wrote a simple Dockerfile which installs all the necessary dependencies and got it to show the help message. However, upon wanting to run the demo, the below stack trace was outputted.

```
WARNING:root:The parameter "result_based_confidence_threshold" not found in OTX_SSD wrapper, will be omitted
Traceback (most recent call last):
  File "/demo/demo.py", line 129, in <module>
    sys.exit(main() or 0)
  File "/demo/demo.py", line 124, in main
    demo = inferencer(models, visualizer)
  File "/usr/local/lib/python3.8/site-packages/otx/api/usecases/exportable_code/demo/demo_package/executors/synchronous.py", line 30, in __init__
    self.converter = create_output_converter(model.task_type, model.labels)
  File "/usr/local/lib/python3.8/site-packages/otx/api/usecases/exportable_code/demo/demo_package/utils.py", line 46, in create_output_converter
    converter_type = task_type_to_label_domain(task_type)
  File "/usr/local/lib/python3.8/site-packages/otx/api/entities/model_template.py", line 254, in task_type_to_label_domain
    TaskType.ACTION_CLASSIFICATION: Domain.ACTION_CLASSIFICATION,
  File "/usr/local/lib/python3.8/enum.py", line 384, in __getattr__
    raise AttributeError(name) from None
AttributeError: ACTION_CLASSIFICATION
```

**Steps to Reproduce**

Like training, I used Docker which I don't think is the issue here.

1. Have a `demo` output from `otx deploy`;
2. `$ docker build -t modeldemo .`
3. `$ docker run --rm modeldemo python /demo/demo.py -h` - works!
4. `$ docker run --rm -v "path/to/model:/mnt/model" -v "path/to/imgs:/mnt/images" modeldemo python -i /mnt/images/something.jpeg -m /mnt/model/ --output /mnt/images/outputs/` - error!

My `/mnt/model` directory consists of `model.xml`, `model.bin`, `config.json` (config is pasted at the bottom of the issue).

**Environment:**

- OS: `python:3.8` (expands to `debian:11`) 
- Framework version: `openvinotoolkit/training_extensions` commit 09a29f5 
- Python version: `python3.8`
- OpenVINO version: `openvino 2022.3.0`
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Dockerfile**:

```dockerfile
FROM python:3.8

RUN apt update && apt install ffmpeg libsm6 libxext6 libgl1-mesa-glx -y

WORKDIR /demo
COPY . /demo
ENV PYTHONPATH=$PYTHONPATH:/demo/model_wrappers

RUN python -m pip install -r requirements.txt && \
    python -m pip install torch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
```

**config.json**:

```json
{
    "type_of_model": "OTX_SSD",
    "converter_type": "DETECTION",
    "model_parameters": {
        "result_based_confidence_threshold": true,
        "confidence_threshold": 0.15000000596046448,
        "labels": {
            "label_tree": {
                "type": "tree",
                "directed": true,
                "nodes": [],
                "edges": []
            },
            "label_groups": [
                {
                    "_id": "6443d9946b4616aebe1b6518",
                    "name": "labels",
                    "label_ids": [
                        "1"
                    ],
                    "relation_type": "EXCLUSIVE"
                }
            ],
            "all_labels": {
                "1": {
                    "_id": "1",
                    "name": "plate",
                    "color": {
                        "red": 24,
                        "green": 121,
                        "blue": 122,
                        "alpha": 255
                    },
                    "hotkey": "",
                    "domain": "DETECTION",
                    "creation_date": "2023-04-22T12:56:51.810000",
                    "is_empty": false,
                    "is_anomalous": false
                }
            }
        }
    },
    "tiling_parameters": {
        "visible_in_ui": true,
        "type": "PARAMETER_GROUP",
        "enable_tiling": false,
        "enable_tile_classifier": false,
        "enable_adaptive_params": true,
        "tile_size": 400,
        "tile_overlap": 0.2,
        "tile_max_number": 1500,
        "header": "Tiling Parameters",
        "description": "Tiling Parameters",
        "_ParameterGroup__metadata_overrides": {},
        "groups": [],
        "parameters": [
            "enable_adaptive_params",
            "enable_tile_classifier",
            "enable_tiling",
            "tile_max_number",
            "tile_overlap",
            "tile_size"
        ]
    }
}
```

**Comment by goodsong81** [comment]
Thank you for the report. AFAIK, the latest develop branch will work well.
It was fixed in #2031. If it's not, @GalyaZalesskaya will help you :)

**Comment by QSmally** [comment]
Ty @goodsong81. It doesn't throw an error after changing the commit hash in `requirements.txt` to `2a05810179bbc49365cc12d6dc36f78e079c13a7`.
